bigtop-packages/src/common/alluxio/do-component-build

```diff
@@ -45,7 +45,7 @@ if [ $HOSTTYPE = "powerpc64le" ] ; then
 
 else
   #need to manually compile the libjnifuse*.so in openEuler
```

Comment:
The comment should be updated too. It would be nice if you can describe the reason why this is needed.

Reply:
@iwasakims Thanks, the comments has updated. https://github.com/apache/bigtop/pull/1229/commits/9f0695218f5a73315101dfac29beef07a80526f5